imap_processing/tests/ultra/unit/test_ultra_l1b_extended.py

```diff
@@ -468,6 +469,25 @@ def test_get_phi_theta(test_fixture):
     np.testing.assert_allclose(theta, expected_theta, atol=1e-03, rtol=0)
 
 
+def test_get_spin_number(test_fixture, use_fake_spin_data_for_time):
+    """Tests that get_spin_number assigns the correct spin number."""
+    df_filt, _, _, de_dataset = test_fixture
+
+    # Simulate a spin table from MET = 0 to MET = 141*15 seconds
+    use_fake_spin_data_for_time(start_met=0, end_met=141 * 15)
+
+    # Convert shcoarse (time) to MET relative to first value
+    # Take off first partial spin.
+    de_met = de_dataset["shcoarse"].values[19::] - de_dataset["shcoarse"].values[19]
+    de_spin = de_dataset["spin"].values[19::]
```

Comment:
It took me a second to figure out what 19 referred to, could you call that a specific variable name here? Also remove the second colon if it isn't used.
```suggestion
    first_spin_index = 19
    de_met = de_dataset["shcoarse"].values[first_spin_index:] - de_dataset["shcoarse"].values[first_spin_index]
    de_spin = de_dataset["spin"].values[first_spin_index:]
```